gridcapa-task-manager-app/src/main/java/com/farao_community/farao/gridcapa/task_manager/app/TaskUpdateNotifier.java

```diff
@@ -6,34 +6,38 @@
  */
 package com.farao_community.farao.gridcapa.task_manager.app;
 
+import com.farao_community.farao.gridcapa.task_manager.api.TaskDto;
 import com.farao_community.farao.gridcapa.task_manager.app.entities.Task;
-import org.springframework.cloud.stream.function.StreamBridge;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.messaging.simp.SimpMessagingTemplate;
 import org.springframework.stereotype.Service;
 
+import java.time.format.DateTimeFormatter;
 import java.util.Set;
 
 /**
  * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
  */
 @Service
 public class TaskUpdateNotifier {
-    private static final String TASK_UPDATED_BINDING = "task-updated";
-    private static final String TASK_STATUS_UPDATED_BINDING = "task-status-updated";
-
-    private final StreamBridge streamBridge;
     private final TaskDtoBuilder taskDtoBuilder;
 
-    public TaskUpdateNotifier(StreamBridge streamBridge, TaskDtoBuilder taskDtoBuilder) {
-        this.streamBridge = streamBridge;
+    @Autowired
```

Comment:
Don't use @Autowired but link it through the constructor

Reply:
link in the constructor